src/google.rs

```diff
@@ -17,15 +18,46 @@ pub struct GoogleResponse {
 }
 
 #[derive(Serialize, Deserialize)]
-pub struct GooglePlayParameters {
+pub struct GooglePlayData {
     pub json: String,
     pub signature: String,
     #[serde(rename = "skuDetails")]
     pub sku_details: String,
 }
 
+impl GooglePlayData {
+    pub fn from(payload: &str) -> std::result::Result<Self, serde_json::Error> {
```

Comment:
why not using our crate `Result`?

Reply:
I don't know if I was just tired or something, but I was getting a missing Deserialize impl error when I used error::Result. Tried deriving Deserialize on Result and it broke the thiserror macros

Reply:
so we can't use our own Result here?

Reply:
Not so far as I can see